e2e/email-sign-up.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { test, expect, type Page } from '@playwright/test';
+import translations from '../client/i18n/locales/english/translations.json';
+
+let page: Page;
+
+test.beforeAll(async ({ browser }) => {
+  page = await browser.newPage();
+  await page.goto('/email-sign-up');
+});
+
+test.afterAll(async () => {
+  await page.close();
+});
+
+test.describe('Email sign-up page', () => {
+
+  test('The page renders with correct title', async () => {
```

Comment:
```suggestion
  test('The page renders with correct title', async ({ page }) => {
```

Because we can get `page` directly from the function args.